core/shared/src/main/scala/matryoshka/implicits/IdOps.scala

```diff
@@ -28,9 +28,9 @@ sealed class IdOps[A](self: A) {
     matryoshka.hyloM(self)(f, g)
 
   object ghylo {
-    def apply[W[_], N[_]] = new Aux[W, N]
+    def apply[W[_], N[_]] = new PartiallyApplied[W, N]
```

Comment:
Renamed these “helper” classes to match the Cats approach, which is named this way to avoid confusion with the _other_ `Aux` pattern. Figured since I was adding new ones, I should make sure the code is consistent.

This is something that multiple implicit lists would make disappear.